spec.html

```diff
@@ -53,19 +53,20 @@ <h1>Intl.Segmenter ([ locales [ , options ]])</h1>
         1. Let _segmenter_ be ? OrdinaryCreateFromConstructor(_newTarget_, *%SegmenterPrototype%*).
         1. Let _requestedLocales_ be ? CanonicalizeLocaleList(_locales_).
         1. Let _opt_ be a new Record.
-        1. Let _matcher_ be ? GetOption(_options_, *"localeMatcher"*, *"string"*, &laquo; *"lookup"*, *"best fit"* &raquo;, *"best fit"*).
+        1. Let _matcher_ be ? GetOption(_options_, `"localeMatcher"`, `"string"`, &laquo; `"lookup"`, `"best fit"` &raquo;, `"best fit"`).
         1. Set _opt_.[[localeMatcher]] to _matcher_.
+        1. Let _strictness_ be ? GetOption(_options_, `"strictness"`, `"string"`,  « `"strict"`, `"normal"`, `"loose"` », `"normal"`).
```

Comment:
UTS35 calls it "Line break style". Historically we kept the options names close to UTS extension key names. If there's no reason to diverge here, maybe we should follow the trend?

http://www.unicode.org/reports/tr35/#Locale_Extension_Key_and_Type_Data

Reply:
Will fix in a follow-on patch.